### ISSUE TITLE
Show % of reviewed features for open & synthetic datasets

### ISSUE THREAD

**Issue by eneerhut** [facebook/Rapid#1611]
### Description

It would be helpful to know the extent to which a dataset we feature has been reviewed by the community. We'd need a way to track when a feature has been added or rejected and some logic for whether one rejection is enough for a feature to be hidden. Once we settle on the logic though, we should have an option to sort data by this % and encourage the community to get to 100%. This is very similar to how the Tasking Manager works.

<img width="1402" alt="image" src="https://github.com/user-attachments/assets/8e4efcf6-73ee-445f-9fd3-039c5176772f">

**Comment by bhousel** [comment]
Agree, if we could get this information from somewhere it would be great to show it to users. 👍
